src/model/data.types.model.ts

```diff
@@ -17,17 +17,16 @@ export enum BeneficialOwnerTypeChoice {
     none = "none"
 }
 
-
 export interface InputDate {
     day: number
     month: number
     year: number
 }
 
 export enum natureOfControl {
-    over25under50Percent = "25",
-    over50under75Percent = "50",
-    over75Percent = "75"
+    over25upTo50Percent = "25",
+    over50upTo75Percent = "50",
```

Comment:
Naming of second one here was fine - "more than 50% and less than 75%" - but now name suggests a value of 75% is allowed. Suggest changing it back to what you had: _over50under75Percent_.